server/src/main/java/org/elasticsearch/persistent/decider/AssignmentDecision.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.elasticsearch.persistent.decider;
+
+import java.util.Locale;
+import java.util.Objects;
+
+/**
+ * {@link AssignmentDecision} represents the decision made during the process of
+ * assigning a persistent task to a node of the cluster.
+ *
+ * @see AssignmentDecider
+ */
+public final class AssignmentDecision {
+
+    public static final AssignmentDecision ALWAYS = new AssignmentDecision(Type.YES);
```

Comment:
That feels like premature optimization.

Reply:
+1 . People can just use yes? I'm not sure what the difference is?

Reply:
Right, it's not required.